pywhat/Data/regex.json

```diff
@@ -2500,6 +2471,26 @@
          "Invalid": []
       }
    },
+   {
+      "Name": "Datadog Client Token",
+      "Regex": "^(pub[a-f0-9]{32})$",
+      "plural_name": false,
+      "Description": "An [#CAE4F1][link=https://docs.datadoghq.com/logs/log_collection/javascript/]Datadog client token[/link][/#CAE4F1]",
+      "Exploit": null,
+      "Rarity": 0.2,
```

Comment:
I feel like you can increase the rarity here.

Reply:
Lets discuss about this in discord.

Reply:
Done. Let's change it to 0.3